clang/test/CAS/mccas-replay.c

```diff
@@ -0,0 +1,23 @@
+// RUN: rm -rf %t && mkdir %t
+
+// RUN: %clang -cc1depscan -o %t/args.rsp  -cc1-args -cc1 \
+// RUN:    -debug-info-kind=standalone -dwarf-version=4 -debugger-tuning=lldb \
+// RUN:    -emit-obj -fcas-backend  -fcas-path %t/cas       %s -o - > /dev/null
+
+// RUN: (cd %t; %clang @%t/args.rsp -o %t/output1.o -Rcompile-job-cache)
```

Comment:
You don't need to `cd %t`, this is used in other tests to test relative search paths.
Also you could only invoke clang once here and check for the "miss" string at the `sed` line below. Invoking again to get a hit is irrelevant for this test.